calendar/internal/config/config.go

```diff
@@ -0,0 +1,10 @@
+package config
+
+// Application holds application configuration values
+type Application struct {
```

Comment:
It looks like an application entity, rather than configuration, at least at this stage.

Reply:
the intention was to have smth like `config.Application`